oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxSensors.java

```diff
@@ -41,30 +41,57 @@
 
     // Base HWMON path, adds 0, 1, etc. to end for various sensors
     private static final String HWMON = "/sys/class/hwmon/hwmon";
+    // Base THERMAL_ZONE path, adds 0, 1, etc. to end for temperature sensors
+    private static final String THERMAL_ZONE = "/sys/class/thermal/thermal_zone";
 
     // Map from sensor to path
     private Map<String, String> hwmonMap = new HashMap<>();
```

Comment:
I want to rename this property - `hwmonMap` - to `sensorsMap`. It sounds a little bit more related than the current name. I also want to remove the use of `this`'s. May I?